bedrock/mozorg/templates/mozorg/rise25/landing.html

```diff
@@ -116,69 +58,46 @@ <h3 class="r25-c-category-title">Advocates</h3>
 
     <div id="artists" class="r25-c-category r25-t-green">
       <div class="r25-c-category-body">
         <h3 class="r25-c-category-title">Artists</h3>
         <p class="r25-c-category-desc">Reimagining AI’s creative potential.</p>
         <p class="r25-c-category-expand">
-          They transcend traditional AI applications, like synthesizing visuals
-          or using large language models. Their projects, whether interactive
-          websites, films, or digital media, challenge our perceptions and
-          demonstrate how AI can amplify and empower human creativity. Their work
-          provokes thought and offers fresh perspectives on the intersection of
-          AI and art.
+          Their work provokes thought and offers fresh perspectives on AI through
+          storytelling across various mediums. Their projects challenge perceptions
+          and show how AI can amplify human creativity. Some also use their craft
+          to challenge the current state of AI and engage in conversations about
+          ensuring technology is applied fairly and equitably.
         </p>
       </div>
-      {% if switch('rise25-nominations-open') %}
-        <p class="r25-c-category-cta">
-          <a class="r25-c-button" href="https://mozilla.formstack.com/forms/rise25" rel="external noopener" target="_blank">Nominate</a>
-        </p>
-      {% endif %}
+
+      <p class="r25-c-category-cta">
+        <a class="r25-c-button" href="https://rise25.mozilla.org/artists/" rel="external noopener" target="_blank">See the honorees</a>
+      </p>
     </div>
 
-    <div id="entrepreneurs" class="r25-c-category r25-t-purple">
+    <div id="builders" class="r25-c-category r25-t-teal">
       <div class="r25-c-category-body">
-        <h3 class="r25-c-category-title">Entrepreneurs</h3>
-        <p class="r25-c-category-desc">Fueling AI’s evolution with visionary ventures.</p>
+        <h3 class="r25-c-category-title">Builders</h3>
+        <p class="r25-c-category-desc">Developing AI through ethical innovation.</p>
         <p class="r25-c-category-expand">
-          These daring individuals are transforming imaginative ideas into reality.
-          They’re crafting businesses and solutions with AI to meet societal needs,
-          improve everyday life and forge new technological paths. They embody
-          innovation, steering startups and projects with a commitment to ethical
-          standards, inclusiveness and enhancing human welfare through technology.
+          They’re the architects of trustworthy AI, including engineers and data
+          scientists dedicated to the principles of open source, open data and
+          open science. They focus on technical proficiency and responsible and
+          ethical construction. Their work ensures AI is secure, accessible and
+          reliable, aiming to create tools that empower and advance society.
         </p>
       </div>
-      {% if switch('rise25-nominations-open') %}
```

Comment:
Do you plan on removing the switch entirely from waffle config, or chances are next year will make use of it again so better to keep it there?;)

Reply:
We can safely remove it from the config once this change has been deployed. Actually there are quite a few redundant switches that we can clean up. I have a work in progress to do that and I'll add this one to the list.

Reply:
The switch is already [off](https://github.com/mozmeao/www-config/blob/fc790851a9a1e3e9ff5980850c5b8d0c0ffbc5f0/waffle_configs/bedrock-prod.env#L84) which is basically the same as if it wasn't there at all.

Why I'm asking is I also keep track of what needs removing https://github.com/mozilla/bedrock/issues/14092#issuecomment-2037115750 and some already removed waiting to clean up: https://github.com/mozmeao/www-config/pull/559 that I'm slowly adding to;) Feel free to build on top of this if it helps…